src/attpc_engine/kinematics/excitation.py

```diff
@@ -73,33 +110,183 @@ def sample(self, rng: Generator) -> float:
         """
         return rng.normal(self.centroid, self.sigma)
 
+    def sample_theta(self, rng: Generator) -> float:
+        """Sample the theta angle distribution of the excitation.
+
+        Parameters
+        ----------
+        rng: numpy.random.Generator
+            The random number generator
+
+        Returns
+        -------
+        float
+            The sampled value
+        """
+        return rng.uniform(self.ang_min, self.ang_max)
```

Comment:
So the thing about polar is that its in a cos(theta) distribution by default. So min and max should be cos(min) and cos(max). We can talk about this one tho, and it will probs be more clear after separating the excitation and angle distributions